src/main/javascript/service/http-interceptor.js

```diff
@@ -7,21 +7,33 @@ angular.module('cat.service.httpIntercept', ['cat.service.message', 'cat.service
  * @name cat.service.httpIntercept:errorHttpInterceptor
  */
     .factory('errorHttpInterceptor', function CatErrorHttpInterceptor($q, $globalMessages, loadingService, $injector) {
+        function toBeIgnored(config) {
+            return !!config && config.skipLoadingService;
+        }
+
         return {
             'request': function (config) {
-                loadingService.start();
+                if (!toBeIgnored(config)) {
+                    loadingService.start();
+                }
                 return config;
             },
             'requestError': function (rejection) {
-                loadingService.stop();
+                if (!toBeIgnored(rejection, rejection.config)) {
```

Comment:
toBeIgnored only takes one parameter, I guess you only meant to pass rejection.config here?